packages/releaser/src/ReleaseWorker/ValidateTildaInComposerJsonReleaseWorker.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker;
+
+use Nette\Utils\Strings;
+use PharIo\Version\Version;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\SplFileInfo;
+use Symplify\MonorepoBuilder\FileSystem\ComposerJsonProvider;
+use Symplify\MonorepoBuilder\FileSystem\JsonFileManager;
+use Symplify\MonorepoBuilder\Release\Contract\ReleaseWorker\ReleaseWorkerInterface;
+
+final class ValidateTildaInComposerJsonReleaseWorker implements ReleaseWorkerInterface
+{
+    /**
+     * @var \Symplify\MonorepoBuilder\FileSystem\ComposerJsonProvider
+     */
+    private $composerJsonProvider;
+
+    /**
+     * @var \Symplify\MonorepoBuilder\FileSystem\JsonFileManager
+     */
+    private $jsonFileManager;
+
+    /**
+     * @var \Symfony\Component\Console\Style\SymfonyStyle
+     */
+    private $symfonyStyle;
+
+    /**
+     * @var bool
+     */
+    private $isSuccessful = false;
+
+    /**
+     * @var string[]
+     */
+    private $lockVersionAllowedPackages = [];
```

Comment:
is this array used (i.e. filled in) somehow somewhere? If I do get its purpose correctly, it should allow me to whitelist some dependencies, right? But how could I do that? Oh, I see - using `lock_version_allowed_packages` in `services.yaml`, right? However, how does this work? It was quite hard to get this :laughing:

Reply:
I see, let us discuss it once more on the phone :wink:

Reply:
Explanation in the comment below is sufficient for me :slightly_smiling_face: